docs/api-reference/uiSchema.md

```diff
@@ -218,5 +232,42 @@ const uiSchema = {
   "ui:title": "Your password"
 };
 ```
+
+## submitButtonProps
+
+Sometimes it's convenient to change the behavior of the submit button for the form. This is the purpose of the `ui:submitButtonProps` uiSchema directive:
+
+By default, this library will set the following options mentioned below for all submit buttons:
+### `hidden` option
+
+You can use this option to hide the submit button
+
+### `disabled` option
+
+You can use this option to disable the submit button
+
+### `className` option
+
+You can use this option to specify a class name for the submit button
+
+### `required` option
```

Comment:
I think we don't need the `required` option because the `hidden` option already accomplishes this purpose?

Reply:
hidden is if you still want to have it in the dom. required removes it completely. Also hidden will only probably work on some browsers.

Reply:
I have removed the hidden option. mentioned in the docs they can pass any other prop they want, so hidden if they want to